src/analysis/normed_space/compact_operator.lean

```diff
@@ -376,55 +371,54 @@ hf
 end continuous
 
 lemma is_closed_set_of_is_compact_operator {𝕜₁ 𝕜₂ : Type*} [nontrivially_normed_field 𝕜₁]
```

Comment:
Since there are quite a few typeclasses to consume, I think a doc string would help the reader see the generality more easily. Maybe:
```suggestion
/-- The set of compact operators from a normed space to a complete topological vector space is closed. -/
lemma is_closed_set_of_is_compact_operator {𝕜₁ 𝕜₂ : Type*} [nontrivially_normed_field 𝕜₁]
```